com.valvesoftware.Steam.metainfo.xml

```diff
@@ -73,33 +73,10 @@
     <release version="1.0.0.62" date="2020-03-19"/>
     <release version="1.0.0.61" date="2019-04-18"/>
   </releases>
-    <content_rating type="oars-1.1">
-    <content_attribute id="violence-cartoon">intense</content_attribute>
-    <content_attribute id="violence-fantasy">intense</content_attribute>
-    <content_attribute id="violence-realistic">intense</content_attribute>
-    <content_attribute id="violence-bloodshed">intense</content_attribute>
-    <content_attribute id="violence-sexual">intense</content_attribute>
-    <content_attribute id="violence-desecration">intense</content_attribute>
-    <content_attribute id="violence-slavery">intense</content_attribute>
-    <content_attribute id="violence-worship">intense</content_attribute>
-    <content_attribute id="drugs-alcohol">moderate</content_attribute>
-    <content_attribute id="drugs-narcotics">moderate</content_attribute>
-    <content_attribute id="drugs-tobacco">moderate</content_attribute>
-    <content_attribute id="sex-nudity">intense</content_attribute>
-    <content_attribute id="sex-themes">intense</content_attribute>
-    <content_attribute id="sex-homosexuality">moderate</content_attribute>
-    <content_attribute id="sex-prostitution">moderate</content_attribute>
-    <content_attribute id="sex-adultery">moderate</content_attribute>
-    <content_attribute id="sex-appearance">intense</content_attribute>
-    <content_attribute id="language-profanity">intense</content_attribute>
-    <content_attribute id="language-humor">intense</content_attribute>
-    <content_attribute id="language-discrimination">intense</content_attribute>
+  <content_rating type="oars-1.1">
     <content_attribute id="social-chat">intense</content_attribute>
-    <content_attribute id="social-info">none</content_attribute>
+    <content_attribute id="social-info">intense</content_attribute>
     <content_attribute id="social-audio">intense</content_attribute>
-    <content_attribute id="social-location">none</content_attribute>
-    <content_attribute id="social-contacts">none</content_attribute>
     <content_attribute id="money-purchasing">intense</content_attribute>
-    <content_attribute id="money-gambling">intense</content_attribute>
```

Comment:
Can collectable cards or something else be considered gambling?

Reply:
Gambling: 
- Mild: Gambling on random events using tokens or credits
- Moderate: Gambling using fictional money
- Intense: Gambling using real money

Reply:
Steam explicitly bans money gambling on its platform. https://store.steampowered.com/eula/471710_eula_0
```
You must not implement any gambling, casino style games, or games of chance that involve any real currency, virtual currency, or Rec Tokens in any way.
```
So any game doing any of this is in violation of Steam policy.

Reply:
As for games, it's tricky. There has been incidents where regulators have considered loot boxes et all as gambling.